lib/keyring_liberator.rb

```diff
@@ -63,7 +63,7 @@ def self.get_all_keyrings
     end
 
     def self.get_keyring_at_path(path)
-      Keyring.from_hash(YAML.load(path.read)) if path.file?
+      Keyring.from_hash(YAML.safe_load(path.read)) if path.file?
```

Comment:
sorry. my local ruby was ​​2.4 . I changed it to ruby ​​2.0 and retry `bundle exec rake spec` and succeeded.